package.json

```diff
@@ -1,13 +1,6 @@
 {
-  "name": "ember-set-helper",
```

Comment:
This name was causing the test-app to resolve `ember-set-helper` to the base of the project – which isn't an ember addon